src/Agent.Sdk/ContainerInfo.cs

```diff
@@ -148,6 +150,19 @@ public List<PortMapping> PortMappings
             }
         }
 
+        public Dictionary<string, string> PathMappings
```

Comment:
Maybe instead of this method, just add an AddPathMapping(string, string) method instead of exposing the entire PathMappings map

Reply:
I have added a method, AddPathMapping and set the mapping via that. :) This getter is just similar to PortMappings.